index.html

```diff
@@ -0,0 +1,211 @@
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+        href="https://fonts.googleapis.com/css2?family=PT+Sans:wght@400;700&family=Sansita+Swashed:wght@600;800&display=swap"
+        rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <nav class="site-nav">
+            <button type="button" class="u-button-reset js-toggle-trigger" aria-controls="site-menu">
+                <span aria-hidden="true" class="menu-icon">
+                    <div class="menu-icon__line"></div>
+                    <div class="menu-icon__line"></div>
+                    <div class="menu-icon__line"></div>
+                </span>
+            </button>
+            <ul id="site-menu" class="u-list-reset menu-list">
+                <li><a href="#details">details</a></li>
+                <li><a href="#speakers">speakers</a></li>
+                <li><a href="#register">register</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <h1>Green-on-Mind Design</h1>
+    <main>
+        <section class="details u-list-reset" id="details">
+            <h2><span aria-hidden="true">-</span> details <span aria-hidden="true">-</span></h2>
+
+            <p><strong>theme:</strong> Designing Sustainably</p>
+            <p><strong>location:</strong> Virtually</p>
+            <p><strong>when:</strong> April 22, 2021</p>
+            <p><strong>time:</strong> 12PM - 3PM (EST)</p>
+            <p><strong>who's invited:</strong> Designers</p>
+            <p><strong>what:</strong> Green-on-Mind Design is proud to present the speakers and date of its next monthly
+                event!
+                As
+                always, it will be virtual
+                to lower attendees' carbon footprint to travel, save money and resources, and to broaden accessibility
+                for
+                anyone wanting to join. It is primarily for designers, as speakers will refer to design terminology, but
+                anyone
+                is welcome. In this conference, we will hear from three speakers: Emma Fanning, Michelle Fehler, and
+                Suhaila
+                Sikand, focusing on ways designers can ensure their practice is more sustainable.</p>
+        </section>
+
+        <section id="speakers" class="speakers">
+            <h2 class="speakerheading"><span aria-hidden="true">-</span> speakers <span aria-hidden="true">-</span></h2>
+
+            <ul class="u-list-reset speaker-section">
+                <li class="speaker-info">
+                    <p class="title">
+                        Staying Green With Clients
+                    </p>
+                    <p class="time">12PM - 12:45PM</p>
+                    <img src="https://cdn.glitch.com/74171c12-683c-4072-8113-bc7e1b6a35cc%2FEmmaFanning%20copy.jpg?v=1607029786475"
+                        alt="close-up of a woman wearing glasses and a turtleneck sweater, leaning against tree"
```

Comment:
This is great `alt` text. Bonus points:
1. Consider capitalizing your `alt` value and treating it like sentence text.
2. To that same end, if you end your `alt` value with a period, a screen reader will pause a beat, treating it just like a sentence.
```suggestion
                        alt="Close-up of a woman wearing glasses and a turtleneck sweater, leaning against tree."
```